webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -139,19 +138,19 @@ class SubscriptionsPage extends Component {
   };
 
   uploadManifest = async (file) => {
-    this.startManifestTask();
+    this.startTask();
     await this.props.uploadManifest(file);
     this.cleanUpManifestTask();
   };
 
   deleteManifest = async () => {
-    this.startManifestTask();
+    this.startTask();
     await this.props.deleteManifest();
     this.cleanUpManifestTask();
   };
 
   refreshManifest = async () => {
-    this.startManifestTask();
```

Comment:
Deleting subscriptions is a manifest task. What were your thoughts when renaming this method?

Reply:
Oh yes! cause we are deleting subscriptions from a manifest. My bad. I renamed it because we are deleting a subscription and not doing any major task on the manifest as a whole. But deleting a subscription changes the manifest. So startManifestTask() makes sense to me. I will rename it back.